tftb/processing/postprocessing.py

```diff
@@ -32,38 +34,10 @@ def hough_transform(image, m=None, n=None):
     if n is None:
         n = ymax
 
-    rhomax = np.sqrt((xmax ** 2) + (ymax ** 2)) / 2.0
-    deltar = rhomax / (m - 1.0)
-    deltat = 2 * np.pi / n
-
-    ht = np.zeros((m, n))
-    imax = np.amax(image)
+    image  =  image[0:m , 0:n]
 
-    if xmax % 2 != 0:
-        xc = (xmax + 1) / 2
-        xf = xc - 1
-    else:
-        xc = xf = xmax / 2
-    x0 = 1 - xc
+    ht, theta, rho = hough_line(image)
```

Comment:
I tried this, too, but its not working. The output of skimage's hough_line might have to be modified a bit. Try out the example here: http://pytftb.readthedocs.org/en/master/auto_examples/plot_5_4_2_hough_noisy_chirp.html#sphx-glr-auto-examples-plot-5-4-2-hough-noisy-chirp-py

We'll have to do something to make the output of `hough_line` match the output of the example.